# QueryFullGraph.srv
# Request
---
# Response
string[] nodes
string[] subject
string[] target
string[] action



